web/server/codechecker_server/session_manager.py

```diff
@@ -368,7 +382,9 @@ def __handle_validation(self, auth_string):
         validation = self.__try_auth_root(auth_string) \
             or self.__try_auth_dictionary(auth_string) \
             or self.__try_auth_pam(auth_string) \
-            or self.__try_auth_ldap(auth_string)
+            or self.__try_auth_ldap(auth_string) \
+            or self.__try_auth_oauth_github(auth_string) \
```

Comment:
We might need a more sophisticated way to decide what method to use for logging in.
When a user tries to log in with github (or google) do we still need to try to login with ldap?

Reply:
now it is just `or self.__try_auth_oauth(auth_string) ` for all aouth providers

Reply:
fixed